frontend/test/unit/specs/utils/attribution-html.spec.ts

```diff
@@ -64,11 +64,11 @@ describe("getAttribution", () => {
 
   it("does not use anchors in plain-text mode", () => {
     document.body.innerHTML = getAttribution(mediaItem, i18n)
-    expect(document.getElementsByTagName("a").length !== 0)
+    expect(document.getElementsByTagName("a")).not.toHaveLength(0)
     document.body.innerHTML = getAttribution(mediaItem, i18n, {
       isPlaintext: true,
     })
-    expect(document.getElementsByTagName("a").length === 0)
+    expect(document.getElementsByTagName("a")).toHaveLength(0)
```

Comment:
The original version of these assertions were noops. `expect()` by itself does nothing. Here's an excellent example of the Jest plugin catching incorrect, false positive (and therefore dangerous because they create false confidence) tests.